Aliases/map - move - over/5eb8d24b-d660-42e5-bbea-47a659bb0827 - Main GVAR.gvar

```diff
@@ -47,7 +58,8 @@ defaultSpells = load_json(get_gvar("d456fdfa-a292-42a1-ab00-b884e79b702f"))
 spellOverlays = defaultSpells.copy()
 # Grab the user selected spells, and update the main dict
 # This allows the user to replace/update items in the original dict
-[spellOverlays.update(load_json(get_gvar(spells))) for spells in load_json(get('mapOverlays','{}')) if get_gvar(spells)]
+userOverlays = load_json(get('mapOverlays','{}'))
+spellOverlays.update(userOverlays)
```

Comment:
So the users mapOverlays was not being loaded in and this was my way of fixing it